taskvine/src/bindings/python3/taskvine.binding.py

```diff
@@ -2209,13 +2293,13 @@ def set_fn_args(self, args=[], kwargs={}):
     ##
     # Specify how the remote task should execute
     # @param self                     Reference to the current remote task object
-    # @param remote_task_exec_method  Can be one of "fork", "direct", or
-    # "thread". Fork creates a child process to execute the function, direct
-    # has the worker directly call the function, and thread spawns a thread to
-    # execute the function
+    # @param remote_task_exec_method  Can be either of "fork" or "direct".
+    # Fork creates a child process to execute the function and direct
+    # has the worker directly call the function.
     def set_exec_method(self, remote_task_exec_method):
-        if remote_task_exec_method not in ["fork", "direct", "thread"]:
-            print("Error, vine_exec_method must be one of fork, direct, or thread")
+        if remote_task_exec_method not in ["fork", "direct"]:
```

Comment:
Code still seems to support the thread method (e.g. q.put).  Do you want to remove it?

Reply:
Matt's PR will be removing it, so I'll wait for his PR to officially remove all the code related to it. This is just a temporary fix that ensures it can't be used.